.github/workflows/test.yaml

```diff
@@ -174,7 +168,7 @@ jobs:
           head -n 3 /home/runner/.tiup/data/$DATA_PATH/pd-0/pd.log
       - name: Build and run backend in the background
         run: |
-          make
+          make package
           make run &
         env:
           UI: 1
```

Comment:
This may override the step runs before but missed the `E2E_TEST: true` env.

```
      - name: Build UI
        run: |
          make ui
        env:
          NO_MINIMIZE: true
          CI: true
          E2E_TEST: true
```